script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -27,19 +27,19 @@ I vaguely imagine a spaceship flying through the dark void of space.
 The noise of the cluttered outside world does not reach me.
 All that's here is my own futile imagination, the sound of my heart pumping blood, and the memories of what had happened just an hour ago.
 <Page9>
-It was out of the blue.
+It was all so out of the blue.//alt suggestion: It was so out of the blue
 'Tohno Makihisa has passed away.
 Tohno Shiki is to return to the Tohno household at once.'
-The telegram from my original home, which I hadn't heard from in nearly seven years, was something to that effect.
+The telegram sent from my original household, whom I hadn't heard from in nearly seven years, said something to that effect.
 <Page10>
-I can't go against a decision made by the main Tohno family. I am still a student, and I rely on them for living expenses.
-Since I felt like being stubborn, I decided to leave the morning of the day I had to return, rather than the night before.
+I couldn't go against a decision made by the main Tohno family. I am still a student after all, and I'm still relying on them for living expenses.
+Since I was being stubborn, I decided to leave the morning of the day I had to return, rather than the night before.//Just not sure about the first bit of "Since I was being stubborn." rephrase maybe?
 <Page11>
-I just wanted to stay as long as possible in the home I had been living in until now―――
-Even though it was my family's decision, it was my way of showing respect to the Arimas, who had welcomed me, a person that they had absolutely no connection with, into their family.
+I just wanted to stay as long as possible in the home I had been living in up until now―――
+Even though it was the head household's order, it was just my way of showing respect to the Arimas, who had welcomed me, a person that they had absolutely no connection with, into their family.
```

Comment:
I don't really like translating 本家"as "head household", it feels unclear. "Main house" is a lot more clear in what it means imo, especially since in far side routes when the Arima household is discussed, 分家 comes up iirc and "head" vs. "branch" doesn't really work.

Moreover, "それは本家の決定とはいえ" is kinda wrong here, "決定" isn't order, it's decision which is slightly different. That said, I'm a little unclear if the decision is his moving to be with the Arimas, him returning, or both.

Here's my re-TL
> Regardless of it being the main house's decision, I was just showing my own form of respect to the Arimas: people who had welcomed me―a person who they had absolutely no connection to――into their family.